src/main/scala/fr/acinq/bitcoin/scalacompat/Transaction.scala

```diff
@@ -57,6 +57,8 @@ object TxIn extends BtcSerializer[TxIn] {
   }
 
   def coinbase(script: Seq[ScriptElt]): TxIn = coinbase(Script.write(script))
+
+  val SEQUENCE_FINAL = fr.acinq.bitcoin.TxIn.SEQUENCE_FINAL
```

Comment:
Is this one really necessary? It isn't cumbersome to directly use the kmp one, is it?

Reply:
Yes but then the use of both `bitcoin` and `bitcoin.scalacompat` namespaces becomes more confusing